internal/distro/fedora/package_sets.go

```diff
@@ -0,0 +1,731 @@
+		panic(fmt.Sprintf("unsupported boot arch: %s", t.Arch().Name()))
+	}
+
+	return ps
+}
+
+// x86_64 Legacy arch-specific boot package set
+func x8664LegacyBootPackageSet(t *imageType) rpmmd.PackageSet {
+	return rpmmd.PackageSet{
+		Include: []string{
+			"dracut-config-generic",
+			"grub2-pc",
+		},
+	}
+}
+
+// x86_64 UEFI arch-specific boot package set
+func x8664UEFIBootPackageSet(t *imageType) rpmmd.PackageSet {
+	return rpmmd.PackageSet{
+		Include: []string{
+			"dracut-config-generic",
+			"efibootmgr",
+			"grub2-efi-x64",
+			"shim-x64",
+		},
+	}
+}
+
+// aarch64 UEFI arch-specific boot package set
+func aarch64UEFIBootPackageSet(t *imageType) rpmmd.PackageSet {
+	return rpmmd.PackageSet{
+		Include: []string{
+			"dracut-config-generic",
+			"efibootmgr",
+			"grub2-efi-aa64",
+			"grub2-tools",
+			"shim-aa64",
+		},
+	}
+}
+
+func qcow2CommonPackageSet(t *imageType) rpmmd.PackageSet {
+	ps := rpmmd.PackageSet{
+		Include: []string{
+			"@Fedora Cloud Server",
+			"chrony",
+			"systemd-udev",
+			"selinux-policy-targeted",
+			"langpacks-en",
+		},
+		Exclude: []string{
+			"dracut-config-rescue",
+			"etables",
+			"firewalld",
+			"geolite2-city",
+			"geolite2-country",
+			"gobject-introspection",
+			"plymouth",
+			"zram-generator-defaults",
+			"grubby-deprecated",
+			"extlinux-bootloader",
+		},
+	}.Append(bootPackageSet(t))
+
+	return ps
+}
+
+func vhdCommonPackageSet(t *imageType) rpmmd.PackageSet {
+	ps := rpmmd.PackageSet{
+		Include: []string{
+			"@core",
+			"chrony",
+			"selinux-policy-targeted",
+			"langpacks-en",
+			"net-tools",
+			"ntfsprogs",
+			"WALinuxAgent",
+			"libxcrypt-compat",
+			"initscripts",
+			"glibc-all-langpacks",
+		},
+		Exclude: []string{
+			"dracut-config-rescue",
+			"geolite2-city",
+			"geolite2-country",
+			"zram-generator-defaults",
+		},
+	}.Append(bootPackageSet(t))
+
+	return ps
+}
+
+func vmdkCommonPackageSet(t *imageType) rpmmd.PackageSet {
+	ps := rpmmd.PackageSet{
+		Include: []string{
+			"@Fedora Cloud Server",
+			"chrony",
+			"systemd-udev",
+			"selinux-policy-targeted",
+			"langpacks-en",
+		},
+		Exclude: []string{
+			"dracut-config-rescue",
+			"etables",
+			"firewalld",
+			"geolite2-city",
+			"geolite2-country",
+			"gobject-introspection",
+			"plymouth",
+			"zram-generator-defaults",
+			"grubby-deprecated",
+			"extlinux-bootloader",
+		},
+	}.Append(bootPackageSet(t))
+
+	return ps
+}
+
+func openstackCommonPackageSet(t *imageType) rpmmd.PackageSet {
+	ps := rpmmd.PackageSet{
+		Include: []string{
+			"@core",
+			"chrony",
+			"selinux-policy-targeted",
+			"spice-vdagent",
+			"qemu-guest-agent",
+			"xen-libs",
+			"langpacks-en",
+			"cloud-init",
+			"libdrm",
+		},
+		Exclude: []string{
+			"dracut-config-rescue",
+			"geolite2-city",
+			"geolite2-country",
+			"zram-generator-defaults",
+		},
+	}.Append(bootPackageSet(t))
+
+	return ps
+}
+
+// fedora iot commit OS package set
+func iotCommitPackageSet(t *imageType) rpmmd.PackageSet {
```

Comment:
this is missing https://github.com/osbuild/osbuild-composer/blob/082da070d16eb6e73c87adcdab30ef1f0d037ccd/internal/distro/fedora33/distro.go#L737

Reply:
Added back, and the package is showing now in the new manifests iot related:
```
      {
        "name": "fedora-release-common",
        "epoch": 0,
        "version": "34",
        "release": "39",
        "arch": "noarch",
        "remote_location": "https://rpmrepo.osbuild.org/v2/mirror/public/f34/f34-aarch64-updates-released-20220113/Packages/f/fedora-release-common-34-39.noarch.rpm",
        "checksum": "sha256:7f6376e3a393b2501d6c38a8b82b8ebc54265e9b4cf3bb54f3e164a9aaf078f0",
        "check_gpg": true
      },
      {
        "name": "fedora-release-identity-iot",
        "epoch": 0,
        "version": "34",
        "release": "39",
        "arch": "noarch",
        "remote_location": "https://rpmrepo.osbuild.org/v2/mirror/public/f34/f34-aarch64-updates-released-20220113/Packages/f/fedora-release-identity-iot-34-39.noarch.rpm",
        "checksum": "sha256:323cf73160530491d2a25119c9b72ee26c0f23863b1bae71f8a04af1f471656b",
        "check_gpg": true
      },
      {
        "name": "fedora-release-iot",
        "epoch": 0,
        "version": "34",
        "release": "39",
        "arch": "noarch",
        "remote_location": "https://rpmrepo.osbuild.org/v2/mirror/public/f34/f34-aarch64-updates-released-20220113/Packages/f/fedora-release-iot-34-39.noarch.rpm",
        "checksum": "sha256:c8ca801cddbe977065fe4d9bf73606f0a14532a74e4276e357698e316b7d4d73",
        "check_gpg": true
      },
```

Reply:
Updated. Please review.